great_expectations/core/expectation_suite.py

```diff
@@ -161,7 +161,6 @@ def __eq__(self, other):
         if not isinstance(other, self.__class__):
             # Delegate comparison to the other instance's __eq__.
             return NotImplemented
-
```

Comment:
@Shinnnyshinshin Why is this blank line being deleted?

Reply:
This was removed by accident... and the fix has now been carried over from the previous PR. Updated :)